File: LeetCode/LeetCode Practice Problems/1217- Minimum Cost to Move Chips to The Same Position.cpp
class Solution {
public:
    int minCostToMoveChips(vector<int>& position) {
        int odd = 0 , even = 0;
        for(auto i:position) {
            if(i%2 == 0) even++;
            else odd++;
        }
        
        if(odd == 0 || even == 0)
            return 0;
        
        return min(odd,even);
    }
};

File: LeetCode/LeetCode Practice Problems/1217-Minimum Cost to Move Chips to The Same Position.java
class Solution {
    public int minCostToMoveChips(int[] position) {
        int even = 0 , odd = 0;
        
        for(int pos: position) {
            if(pos%2 == 0) even++;
            else odd++;
        }
        
        return Math.min(even , odd);
    }
}
